platform/persistence/platform-persistence-commands/src/test/java/org/codice/ddf/persistence/commands/StoreImportCommandTest.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.persistence.commands;
+
+import static org.hamcrest.Matchers.hasEntry;
+import static org.hamcrest.Matchers.hasSize;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.verify;
+
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import org.codice.ddf.persistence.PersistenceException;
+import org.codice.ddf.persistence.PersistentStore;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class StoreImportCommandTest {
+  @Mock private PersistentStore persistentStore;
+
+  @Captor private ArgumentCaptor<List<Map<String, Object>>> itemsCaptor;
+
+  private final StoreImportCommand command = new StoreImportCommand();
+
+  @Before
+  public void setUp() {
+    command.persistentStore = persistentStore;
+  }
+
+  @Test
+  public void testImportFile() throws PersistenceException {
+    command.type = "metacard";
+    command.filePath = getClass().getResource("/test.json").getPath();
+    command.storeCommand();
+
+    verify(persistentStore).add(eq("metacard"), itemsCaptor.capture());
+
+    final List<Map<String, Object>> items = itemsCaptor.getValue();
+    assertThat(items, hasSize(1));
+
+    final Map<String, Object> item = items.get(0);
+    assertThat(item, hasEntry("test_bin", new byte[] {102, 111, 111, 98, 97, 114}));
```

Comment:
how do we get these byte array values? I'm probably just losing it in translation looking at the test_bin field.

Reply:
That field is a Base64-encoded string. The decoded value is `foobar` and these are the ASCII byte values of those letters. I can add a comment to explain that.

Reply:
Done